style.css

```diff
@@ -21,6 +24,10 @@ nav ul {
   display: none;
 }
 
+.main-nav:hover ul {
```

Comment:
You have this code - it means when you hover over to that `.main-nav` class, the `ul` will be displayed.
So you don't need to have
```
 $('.icon-menu').on('click', function() {
   $('.main-nav').toggleClass( );
});
```
in your `app.js`